optimade/server/query_params.py

```diff
@@ -224,15 +224,13 @@ def __init__(
             description="RECOMMENDED for use with _cursor-based_ pagination: using `page_cursor` and `page_limit` is RECOMMENDED.",
             ge=0,
         ),
-        page_above: int = Query(
-            0,
+        page_above: str = Query(
+            "",
             description="RECOMMENDED for use with _value-based_ pagination: using `page_above`/`page_below` and `page_limit` is RECOMMENDED.\nExample: Fetch up to 100 structures above sort-field value 4000 (in this example, server chooses to fetch results sorted by increasing `id`, so `page_above` value refers to an `id` value): `/structures?page_above=4000&page_limit=100`.",
```

Comment:
```suggestion
            description="RECOMMENDED for use with _value-based_ pagination: using `page_above`/`page_below` and `page_limit` is RECOMMENDED.\nExample: Fetch up to 100 structures above sort-field value 4000 (in this example, server chooses to fetch results sorted by increasing `id`, so `page_above` value refers to an `id` value): `/structures?page_above="4000"&page_limit=100`.",
```

Reply:
This is direct from the spec, I am wondering what we really meant here at the time.

Reply:
See Rickard's reponse at https://github.com/Materials-Consortia/OPTIMADE/issues/458, he has convinced me that this is fine and doesn't need quotes

Reply:
I think I understand what was meant with the spec, but there are at least two things to improve in the example.
1 There should be a sort query parameter in the request. 
2 If they are using the id to sort on, the value of should be a string "4000" as the type of id is string.

You can only use the page_above / below in combination with a sort query parameter. 
The value of page_above indicates the value of the property on which you sorted, from which you want to return values. 
The value of page_above and page_below can thus take any sortable type. 

An advantage of this pagination method is that you can't get double entries, in case new entries are added to the database in between requests.
I think the OPTIMADE spec should be updated to clarify the example.

Reply:
Okay, lets continue the discussion there. For now I will merge this I think (after fixing your other suggestion on docstrings) -- then I will start work on piping these parameters to elasticsearch.